src/logic/safe/store/models/types/transactions.d.ts

```diff
@@ -35,12 +35,12 @@ export enum SettingsChangeMethods {
 // note: this extends SAFE_METHODS_NAMES in /logic/contracts/methodIds.ts, we need to figure out which one we are going to use
 export type DataDecodedMethod = TransferMethods | SettingsChangeMethods | string
 
-export interface DecodedValue {
+export interface ValueDecoded {
   operation: Operation
```

Comment:
Operation type updated. 

Instead 
` {
  CALL = 'CALL',
  DELEGATE_CALL = 'DELEGATE_CALL',
  CREATE = 'CREATE',
}`

Should be type Number

Reply:
Should be ok now :)